geonode/harvesting/api/serializers.py

```diff
@@ -147,8 +147,8 @@ def validate(self, data):
         if worker_type is not None and worker_config is not None:
             try:
                 utils.validate_worker_configuration(worker_type, worker_config)
-            except jsonschema.exceptions.ValidationError as exc:
-                logger.exception(exc)
+            except jsonschema.exceptions.ValidationError:
+                logger.exception()
```

Comment:
This is still going to fail, because `logger.exception()` must take a single `msg` argument, as stated in the docs:

https://docs.python.org/3.8/library/logging.html#logging.Logger.exception

Sorry for the trouble :sweat_smile:

Reply:
Removed